### ISSUE TITLE
Link Issues - Not keyboard accessible and no accessible name - ANDI

### ISSUE THREAD

**Issue by csgreer6** [religious-ecologies/website#26]
Doing an ANDI check on the omeka page of Religious Ecologies using Google Chrome on a Mac - I am mostly discovering link issues.

Seeing issues on the homepage and the items page. Screenshots are attached. 
![Screen Shot 2020-10-29 at 12 21 53 PM](https://user-images.githubusercontent.com/70596028/97605217-c39cc080-19e4-11eb-865a-4f2c0e8358db.png)
![Screen Shot 2020-10-29 at 12 37 49 PM](https://user-images.githubusercontent.com/70596028/97605249-cb5c6500-19e4-11eb-8a8d-474020e03e11.png)
![Screen Shot 2020-10-29 at 12 42 03 PM](https://user-images.githubusercontent.com/70596028/97605273-d31c0980-19e4-11eb-8067-44a32d4fb05c.png)
![Screen Shot 2020-10-29 at 12 42 13 PM](https://user-images.githubusercontent.com/70596028/97605288-d7482700-19e4-11eb-9b32-df6955eb6f15.png)

**Comment by hepplerj** [comment]
This appears completed. Closing out.
